### ISSUE TITLE
Support transfer of Avatars

### ISSUE THREAD

**Issue by LaurentSalou** [mvs-org/lightwallet#333]
It should be possible to transfer an Avatar from an address to another

**Comment by vorapoap** [comment]
Totally agree!!!
Is feature already available and just needed to be activated on the lightwallet?

**Comment by canguruhh** [comment]
@vorapoap yes its activated on the consensus layer but its not supported on the wallet. There are some issue with loading the avatar information because currently it loads the avatar info from the issuance transaction which it expects to find in your transaction history. If you received the avatar from a different address then we need to do an additional step to get this info. We are working on that
